management/templates/index.html

```diff
@@ -383,6 +390,11 @@ <h4 class="modal-title" id="errorModalTitle"> </h4>
   else if (typeof localStorage != 'undefined' && localStorage.getItem("miab-cp-credentials"))
     api_credentials = localStorage.getItem("miab-cp-credentials").split(":");
 
+  if (api_credentials[0].length !== 0) {
+    $("#guestNav").addClass("hidden");
```

Comment:
It is not sure that we have a working login if they are set. The box can be restarted in the main time and then you need to login again.

I would suggest to either move this two items down, or maybe you can add something to the `show_login` function to show the guestNav if it is called.